code/modules/jobs/job_types/bridge.dm

```diff
@@ -71,6 +72,10 @@ var/list/posts = list("weapons", "helms")
 		H.job = "Helms Officer"
 		W.update_label(newjob = H.job)
 
+	var/obj/item/device/pda/P = H.belt
+	if(P)
```

Comment:
I'm working on another PR at the moment, specifically this file, just merge this PR like this, and I'll change it to an istype in the other PR.